### ISSUE TITLE
protein coding product name that contain 'tRNA' are not getting proper mRNA/CDS tbl

### ISSUE THREAD

**Issue by hyphaltip** [nextgenusfs/funannotate#47]
Not sure if this is a GAG issue or funnotate yet but these are .tbl features for protein coding genes. This is only happening for genes with tRNA in the product name so there must be a regex somewhere catching these. Maybe a GAG issue but flagging it here for starters.

```
266241	267371	gene
			locus_tag	B0A48_09466
266241	267371	mRNA
			product	tRNA N6-adenosine threonylcarbamoyltransferase
266241	267371	CDS
			product	tRNA N6-adenosine threonylcarbamoyltransferase
			transl_table	1
			protein_id	lcl|scaffold_21_1
			note	EggNog:ENOG410PG1E~BUSCO:EOG092D2P7K
			db_xref	PFAM:PF00814
```
It should look like
```
266241	267371	gene
			locus_tag	B0A48_09466
266241	267371	mRNA
			product	tRNA N6-adenosine threonylcarbamoyltransferase
			protein_id	gnl|ncbi|B0A48_09466-T1
			transcript_id	gnl|ncbi|B0A48_09466-T1_mrna
266241	267371	CDS
			product	tRNA N6-adenosine threonylcarbamoyltransferase
			protein_id	gnl|ncbi|B0A48_09466-T1
			transcript_id	gnl|ncbi|B0A48_09466-T1_mrna
			transl_table	1
			protein_id	lcl|scaffold_21_1
			note	EggNog:ENOG410PG1E~BUSCO:EOG092D2P7K
			db_xref	PFAM:PF00814
```

**Comment by nextgenusfs** [comment]
I think this is a bug in GAG that then I tried to fix and my regex must not be perfect.  The original problem was that tRNA genes were being treated as protein coding from GAG, so I have a cleanup step that is trying to fix the tbl file.  So looks like I just need to fix that regex that is identifying the tRNA genes and deleting the following line.

**Comment by nextgenusfs** [comment]
Probably would have been easier to change this in GAG, I will look into that or change the regex search slightly.

**Comment by nextgenusfs** [comment]
https://github.com/nextgenusfs/funannotate/commit/8ab746ebdd2ee86e0f57474091f01131182ab42e this is perhaps a short term fix.  But the search is looking for the following in the tbl file, adding the hyphen makes it work on the genomes that I have.  Can you check on your table file to see if this will work?  I only want to pick up tRNA products.
```
#this is old way (except not in python), picked up a few it shouldn't have
cat Pseudogymnoascus_destructan.tbl | grep '\t\t\tproduct\ttRNA'

#fixed version adding hyphen is specific enough on the few test cases i have in front of me
cat Pseudogymnoascus_destructan.tbl | grep '\t\t\tproduct\ttRNA-'
```

**Comment by nextgenusfs** [comment]
fixed in v0.6.0
